### PR TITLE
[Telink]: BLE commissioning (docker part)

### PR DESCRIPTION
**Pull request by rikorsev**
#### Problem
Currently Zephyr OS does not contain implemented Telink BLE controller yet.
So that, BLE functionality for Matter project is implemented with Telink BLE SDK.
This PR contains update of docker image with Zephyr that adapt Telink Radio module work in 
both BLE and Thread modes (not concurrently). 
The PR is closely related to: https://github.com/project-chip/connectedhomeip/pull/15723

#### Change overview
Updates in Telink docker image that contains late Zephyr source code with modifications in Telink radio driver

#### Testing
Tested manually.
Steps: 
* Build image
* Run docker
* Check if Telink lighting-app example able to be built successfully

### CODE REVIEW

File: integrations/docker/images/chip-build-telink/Dockerfile [hunk 1]
```diff
@@ -11,33 +11,37 @@ RUN set -x \
...
+    && /tmp/zephyr-toolchain-riscv64-setup.run -- -d /opt/telink/zephyr-sdk-0.13.2 \
     && : # last line
 
 # Setup Zephyr
-ARG ZEPHYR_REVISION=19c519eb05d6d24a23fd084a3ab6c1a78edf5536
+ARG ZEPHYR_REVISION=b522ea774ab31c3e636f25da374b43855e9d4e60
 WORKDIR /opt/telink/zephyrproject
 RUN set -x \
     && python3 -m pip install -U --no-cache-dir \
     west==0.11.1 \
```
Comment (Damian-Nordic) [inline integrations/docker/images/chip-build-telink/Dockerfile:24]: you can probably bump the west version by the way

File: integrations/docker/images/chip-build-telink/Dockerfile [hunk 2]
```diff
@@ -11,33 +11,37 @@ RUN set -x \
...
-    && git clone https://github.com/zephyrproject-rtos/zephyr \
+    && git clone https://github.com/rikorsev/zephyr \
     && cd zephyr \
     && git reset ${ZEPHYR_REVISION} --hard \
     && west init -l \
     && cd .. \
     && west update \
```
Comment (Damian-Nordic) [inline integrations/docker/images/chip-build-telink/Dockerfile:30]: newer west supports `-o=--depth=1` for shallow cloning

**Review by Damian-Nordic — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]
Fast tracking platform changes.

**Review by yunhanw-google — APPROVED** [review]

**Comment by andy31415** [comment]
> Hi @Damian-Nordic, @woody-apple
> 
> As I see **doxygen** and **infenion** docker images have some troubles to be built. Could you tip me, what shell I do regarding to this?
> 
> Regards, Alex

Infineon was fixed in 0.57 just today. For doxygen I will have to review. Generally when image `A` is updated, I merge even if some other image fails since they are independent. Unfortunately docker images depend on 3rd party things that change (e.g. linux package versions or various download URLs)

**Comment by rikorsev** [comment]
@andy31415 

doxygen and infineon images are fine now. But each time after rebase I face non stable issues with other CI jobs, like:

- Build example - Linux Standalone / Linux Standalone (pull_request) - Reason: timeout
- Builds / Build on Linux (fake, gcc_release, clang, mbedtls, simulated) (pull_request) - Reason: cancellation
- Builds / Build on Darwin (clang, python_lib, simulated) (pull_request)

It seems that this jobs are not stable. May we do some with it?

Regards,
Alex

**Pull request merged**